pkg/operator/operator.go

```diff
@@ -164,6 +164,7 @@ func NewOperator(ctx context.Context, operator *operator.Operator) (context.Cont
 		subnetProvider,
 		unavailableOfferingsCache,
 		pricingProvider,
+		operator.GetClient(),
```

Comment:
Is this useable or should I move the client into the controller setup in pkg/controllers/providers/instancetype/controller.go?